pkg/storkctlcli/storkctlcli.go

```diff
@@ -0,0 +1,161 @@
+package storkctlcli
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/libopenstorage/stork/pkg/storkctl"
+	"github.com/portworx/sched-ops/task"
+	"github.com/sirupsen/logrus"
+
+	"github.com/portworx/torpedo/pkg/aetosutil"
+	"github.com/portworx/torpedo/pkg/log"
+)
+
+var dash *aetosutil.Dashboard
+
+const (
+	drPrefix = "automation-"
+	actionRetryTimeout = 10 * time.Minute
+	actionRetryInterval = 10 * time.Second
+)
+
+var (
+	migSchedNs = "kube-system"
+)
+
+func ScheduleStorkctlMigrationSched(schedName, clusterPair, namespace string, extraArgs map[string]string) error {
+	if namespace != "" {
+		migSchedNs = namespace 
+	}
+	cmdArgs := map[string]string{
+		"cluster-pair": clusterPair,
+		"namespace":   migSchedNs,
+	}
+	err := createMigrationScheduleCli(schedName, cmdArgs, extraArgs)
+	return err
+}
+
+func createMigrationScheduleCli(schedName string, cmdArgs map[string]string, extraArgs map[string]string) error {
+	factory := storkctl.NewFactory()
+	var outputBuffer bytes.Buffer
+	cmd := storkctl.NewCommand(factory, os.Stdin, &outputBuffer, os.Stderr)
+	migCmdArgs := []string{"create", "migrationschedule", schedName}
+	// add the custom args to the command
+	for key, value := range cmdArgs {
+		migCmdArgs = append(migCmdArgs, "--"+key)
+		if value != "" {
+			migCmdArgs = append(migCmdArgs, value)
+		}
+	}
+	if extraArgs != nil {
+		for key, value := range extraArgs {
+			migCmdArgs = append(migCmdArgs, "--"+key)
+			if value != "" {
+				migCmdArgs = append(migCmdArgs, value)
+			}
+		}
+	}
+	cmd.SetArgs(migCmdArgs)
+	// execute the command
+	logrus.Infof("The storkctl command being executed is %v", migCmdArgs)
+	if err := cmd.Execute(); err != nil {
+		if err != nil {
+			return fmt.Errorf("Error in executing create migration schedule command: %v", err)
+		}
+	}
+	// Get the captured output as a string
+	actualOutput := outputBuffer.String()
```

Comment:
Can delete this as we don't need the output unless it has failed.

Reply:
Done.